### DISCUSSION TITLE
너무 극도로 공격적인 카드풀

### DISCUSSION THREAD

**Discussion by 115dkk** [scarf005/Marisa#104] [Balance]
영어로 뭐 써보려고 했는데 잘 안돼서 한글 씀. 존댓말 쓸까 했지만 디시에서 보고 넘어온거니 음슴체 쓰도록 하겠습니다

마리사 카드들 알다시피 죄다 공카에 집중되어 있고 방카는 너무 심각하게 후달림

일단 방카 카드풀이 20%밖에 없다는 건 유명한 거고

희귀카드에 방어 계열 카드는 딱 하나, 그나마도 카드 버린 만큼 방어도 4(6) 들어오는 '토큰'을 던져주는 엄청나게 애매한 카드

글고 그 외에 방카들을 보면

충전된 에너지 쓰면 방어 6(9) 주는거

강화시 판금 5(8) 주는거

약화 99

이 세개는 후반까지도 쓸만하지만 나머지가 문제임 (6A는 거의 쓴 적 없으니 제외하는거 양해해주셈)

우선 충전 에너지만큼 방어하는 카드는 정말 작정하고 에너지만 쌓는거 아니면 쓰기 힘듬. 스택은 그래도 카드 집다보면 사서가 되니까 효과 볼 만 한데 얘는 자연스럽게 충전 쓰면 방어도 날아감

목성은 그래도 가성비는 되게 봐줄만하지만 결국 깡방어 카드라 사일도 아닌 놈이 그거에 의존하기엔 불안정함

디버프 제거해주는 카드는 소멸되니까 지속 방어에는 부적합, 그나마도 흘려보내기 방어도인데 드로 없음

2코 방카는 깡방어도가 12(14)인데 노강은 와쳐 보호와 같고 강화는 보호보다 구림

이콜로지컬은 힘깎 솔직히 많지 않은데 그나마도 이번 턴만 적용이라 방카로서는 불안정함. 덤으로 무작위임

소행성대는 럭키 구르기임. 그래도 마냥 나쁘진 않은데 깡방어도고, 특정 파워를 안키면 가성비가 나쁘다는게 많이 아쉬움

잠깐의 휴식과 은하수는 좆비 방어도니 생략

이러니까 고승천은

A. 2막 가서 제대로 된 딜 플랜 마련 못하고 파킨
B. 1,2,3, 창방까지 딜찍누 하는데 마지막에 심장가서 사망

이런 일이 비일비재함

그래서 내 생각에는

A. 몇개의 카드를 방어 카드로 리워크하거나 (특히 희귀 카드)
B. 잠깐의 휴식, 은하수, 이콜로지컬 등등 버프

특히 마리사 방카풀이 너무 좁아서 겜 내에서 방카를 제대로 볼 수 있을지 알 수 없는 점부터가 문제니 A는 조금이라도 반영되었으면 좋겠음. 주인장은 어떻게 생각하심?

모드캐는 아클 사일 디펙만큼 많이 하지는 않기 때문에 오류가 있을 수 있음. 그런 거 있으면 지적해주심 특히 감사하겠습니다

**Comment by scarf005** [comment]
우선 의견 감사감사
아직 총 플레이시간이 50시간도 안된 슬알못이어서 지금 내는 아이디어가 밸런스 조절이 미숙한거는 양해바람
모두 다 이렇게 바꾸겠다는 거는 아니고 개별 아이디어니까 편하게 봐주셈
이상하거나 너무 사기다싶으면 의견 내주시면 감사감사

## 버프
직업 특성인 증폭, 충전 시너지, 뽑을 카드 불리기를 중점으로 고려해봄

### 6A
<details>
  <summary>사진</summary>
  <img src="https://user-images.githubusercontent.com/54838975/211176128-42341b49-990d-491a-97e8-0423bedecce3.png" alt="image" />
</details>
언럭키 철의파동임. 돌진만큼의 이득을 줘야됨

#### 개선안
- 피해를 5(7) 줍니다 -> 피해를 충전량 + 3(6) 만큼 줍니다

### 은하수
<details>
  <summary>사진</summary>
  <img src="https://user-images.githubusercontent.com/54838975/211176368-34108a0b-b01d-4879-acd6-b42c338b15e1.png" alt="_02" />
</details>

1코여서 전체적인 성능이 너무 나쁨
그냥 럭키수비임
#### 개선안
방어도 5(7) -> 7(9)

### 매직 앱소버
<details>
  <summary>사진</summary>
  <img src="https://user-images.githubusercontent.com/54838975/211175136-c6c8fa45-ef29-45a2-a409-69ff547a0809.png" alt="image" />
</details>

### 개선안
> 소멸 안함
방어도를 8 얻습니다
내게 해로운 효과를 부여하려 하는 적의 수만큼 방어도를 추가로 8 얻습니다
증폭 E: 디버프를 하나 제거하고 방어도를 8 얻습니다

### 에너지 반동
<details>
  <summary>사진</summary>
  <img src="https://user-images.githubusercontent.com/54838975/211176866-b7564760-1c28-4be7-853d-d0c2bc9d38c6.png" alt="_06" />
</details>

#### 개선안
> 충전량 x2만큼 방어도를 얻습니다
고갈을 1 얻습니다
강화시: 보존

## 은하 헤일로
<details>
  <summary>사진</summary>
  <img src="https://user-images.githubusercontent.com/54838975/211177078-4faf5dae-1c34-4759-ac65-e125a223e5af.png" alt="_01" />
</details>

흐릿함과 비슷하게 바꿔봄

#### 개선안
> 특별로 변경, 가격: 1E

방어도를 10(14) 얻습니다.
충전을 3(4) 얻습니다
증폭 E: 이번 턴 방어도가 사라지지 않습니다

### 목성형 행성
<details>
  <summary>사진</summary>
  <img src="https://user-images.githubusercontent.com/54838975/211176840-2e492014-5666-4a1f-8d4d-5a1fc04fb149.png" alt="_03" />
</details>

#### 개선안
> 증폭 E: 방어도를 10(14) 추가로 얻습니다. 뽑을 카드 더미에 화상 카드를 1개 넣습니다.

초신성과 시너지를 좀 더 줌

### 잠깐의 휴식
<details>
  <summary>사진</summary>
  <img src="https://user-images.githubusercontent.com/54838975/211175974-72cf4c3b-f7e1-420e-9ce1-edbef24c1be5.png" alt="_07" />
</details>

충전스택 유지하려고 쓰는거여서 강화 안하면 증폭 못쓰는 수비임. 충전 유지하는건 이점이지만 증폭 못쓰는건 심한 페널티니까 좀더 이득을 줘도 될듯

#### 개선안
방어도를 5(7) 얻습니다 -> 방어도를 10(15) 얻습니다

### 소행성대
<details>
  <summary>사진</summary>
  <img src="https://user-images.githubusercontent.com/54838975/211177516-9302591a-127b-461c-a312-f83df2af6c94.png" alt="_13" />
</details>

#### 개선안
방어도 8(11) -> 10(14)

확실히 얘는 그대로 놔도 될것같긴 함

### 딥 이콜로지컬 봄
<details>
  <summary>사진</summary>
  <img src="https://user-images.githubusercontent.com/54838975/211174882-dc408974-936d-4215-ba5a-6119bf48e02f.png" alt="image" />
</details>

#### 개선안 1
> 가격: 1
무작위 적에게 피해를 5(7) 주고 힘을 3(4) 감소시킵니다.
증폭 E: 적의 수만큼 반복합니다

#### 개선안 2
> 가격: X
무작위 적에게 피해를 5(7) 주고 힘을 3(4) 감소시킵니다.
X(X + 2)번 반복합니다.

### 오러리즈 선
<details>
  <summary>사진</summary>
  <img src="https://user-images.githubusercontent.com/54838975/211175033-4d805eb2-6332-41b0-8fd0-35a57f5e9df6.png" alt="_08" />
</details>

전투시에 충전을 자주 발동하는게 힘들어서 충전 쌓는 전략으로 가면 사실상 봉인인데 발동 조건을 증폭으로 바꾸는게 어떨까 싶음
#### 개선안 1
- 충전이 소모될 때마다 방어도를 6(9) 얻습니다 -> 증폭을 사용할 때마다  방어도를 6(9) 얻습니다

#### 개선안 2
- 희귀로 변경
- `E 1` -> `E 2`
- 충전이 소모될 때마다 방어도를 6(9) 얻습니다 -> 증폭을 사용할 때마다  판금 갑옷을 1(2) 얻습니다

## 방어 카드로 리워크
### 암흑 물질
<details>
  <summary>사진</summary>
  <img src="https://user-images.githubusercontent.com/54838975/211177672-fc159d9d-6d3c-4753-99b8-772d5c4111e1.png" alt="_05" />
</details>

증식하는 수비면 수비보다 안 좋은 것 아닌?가? 굳이 시너지를 따져도 새틀라이트 일루전밖에 없음

#### 개선안
> 가격: E0
휘발성, 사용 불가. 소멸될 때마다 버린 카드의 수만큼 방어도를 얻습니다.
증폭 E: 이 카드의 복사본 2장을 뽑을 카드 더미에 넣습니다.

새틀라이트 일루전 시너지 추가 + 카드 도배 방지

### 준비운동
<details>
  <summary>사진</summary>
  <img src="https://user-images.githubusercontent.com/54838975/211177763-08d6c8d3-4dcc-4d2d-8df1-e7429ec97bec.png" alt="image" />
</details>

#### 개선안
라이징 스윕 대신 6A 주기?


### 궤도
<details>
  <summary>사진</summary>
  <img src="https://user-images.githubusercontent.com/54838975/211177316-0213373c-ad5a-4f80-9050-9d6041965472.png" alt="image" />
</details>

소멸하기 전까지는 그냥 손패수 -1임

#### 개선안
- 카드를 뽑을 시 방어도 3(5) 추가

###  얼티밋 쇼트웨이브
<details>
  <summary>사진</summary>
  <img src="https://user-images.githubusercontent.com/54838975/211177270-51b32915-2bd8-477d-9c6f-3a7a1259ffbb.png" alt="image" />
</details>

이득을 보려면 덱을 자주 교체해야하는데 마리사 덱 특성상 덱이 클수록 유리해지는 카드가 많아서 생각보다 별로인 것 같음

#### 개선안 1
- 방어도 5(7) 추가로 획득

#### 개선안 2
- 효과를 1 증가 -> 이 효과를 1번 더 반복

### 파이널 스파크
<details>
  <summary>사진</summary>
  <img src="https://user-images.githubusercontent.com/54838975/211175174-af22249e-0718-4f93-9e9c-2f770c6dd1e8.png" alt="_11" />
</details>

이것도 희귀 카드인데 한번도 써본 적이 없어서 이유를 생각해봤는데
1. 발동 조건이 매우 까다로움
2. 충전 스택 붙은 공격카드 쓰는게 공격력은 비슷한데 쓰기는 훨씬 쉬움

#### 개선안
> 가격: 7E(5E)
보존, 불가침 1.
모든 적에게 피해를 40 주고 턴을 마칩니다.
'스파크' 카드를 사용할 때마다 비용이 1 감소합니다.

원본 슈팅게임에서는 이 기술이 전멸폭탄이어서 [무적시간이 긴](https://youtu.be/tCwNVHia_E4?t=1489) 점을 반영해봄

**Reply by 115dkk** [reply]
일단 ㅁㅇ 좀 마이 늦었음

6A: 괜찮은 안이긴 한데 언럭키 철파가 아니라 럭키 철파라고 봄. 실제로 저거와 정확히 똑같은 카드로 철파 교체하자고 갤에 제안할 때 상향이라는 주장이 더 많았음

물론 충전이 있는 대신 힘 얻기 힘든 마리사인 만큼 샹항 필요한 건 팩트고 님이 말한 대로면 충분한 상향이라 보임

은하수: 동의함

매직 앱소버: 글쎄... 디버프 하나 지우는 건 매우 큰 메리트인데 소멸이 안되면서 계속 쓸 수 있다면 밸붕의 우려가 매우 큼. 나는 디버프 제거 컨셉을 포기하는 대신 증폭 효과를 방어적으로 더 강하게 주는 게 ㄱㅊ을 거 같음.

에너지 반동: 굿 아이디어

은하 헤일로: 특카임을 감안해도 코스트 대비 방어도가 너무 높음. 저러면 가뜩이나 사기인 증폭 무한 카드가 더 사기임. 코스트 하나 올리고 방어도를 더 올리는 것을 권장함.

거대 기체 행성: 좋은 아이디어

잠깐의 휴식: 강화하면 좋은 효과가 붙는데 방어도를 너무 많이 올리면 밸붕의 소지가 매우 큼. 10(13) 정도면 충분함.

만약 강화 방어도가 반드시 저러면 좋겠다! 라면 차라리 화끈하게 13(15) 같은 식으로 노강이 더 쎄져야 함. 강화 한번에 5방 오르는 건 매우 큰 변화임

소행성대: 내 생각엔 9(12) 정도면 좋지 않을까? 아예 10까지 올려버리면 디펙트 운다

힘깎: ~번 줄입니다 이거 뭐임? '번' 빼고 3, 4 줄인다 이런 식으로 본다면, 나는 1번이 나아보임. 무작위에 모든 에너지를 투자하는 미친놈은 매우 적을거라 생각함

오러리즈 썬: 2번은 덱 굴러가는게 기괴해질 거 같은데. 슬더스가 판금갑옷이나 금속화를 짜게 주는 건 다 이유가 있음. 1번은 자칫하면 아클의 타락무감같은 것을 소멸도 없이 만드는 꼴이 될 수 있어서 좀 그럼. 마리사 이 새끼 증폭 무한 발동 파워 있는거 잊지 마셈

암흑 물질: 증폭에 사용 불가 속성이 있는 이유를 잘 모르겠음... 정확히는 의도하는 대로 작동할지 의문임. 그냥 1코 해도 될 거 같은데. 스택처럼 만드는 아이디어는 아주 좋다고 생각함

근대 준비운동 짤은 뭐냐

궤도: 뽑을 때 방어도 좆비만큼 얻는 건 큰 이득이 아닐 뿐더러, 그렇잖아도 전략가나 반사신경마냥 소멸하면 고밸류를 보여주는 카드임. 마리사 소멸 카드는 꽤 있는 편이라 소멸하지 않을 때를 대비한 조금의 이득을 만드는 것은 카드를 난잡하게 만들 뿐임

얼티밋 쇼트웨이브: 저게 원작에서 뭐하는 물건인지 모르겠는데, 혹시 설명 가능함? 나는 저게 아예 증폭 컨셉 제거하고 에너지 펌핑 딸린 방카로 보직 변경했으면 슬더스적으론 좋을 거 같은데 동방적으로는 몰?루라

파이널 스파크: 대부분 정답인데, 특히 모든 딜뻥을 오지게 받는 머신건 스파크나 충전 딜뻥이 미치광이 수준인 엘리스 선물 같은 씹사기 공카가 있어서 밀리는게 매우 큼. 지속딜은 막말로 준비운동 한두개 챙기고 화싱공격 죽창만 잘 날려도 20승천까지 충분함. 단 스파크 유물 먹고 스파크 집중으로 가면 스파크 자체가 부족해서 못쓰는 일은 흔치 않음 그냥 쓰기도 전에 전투가 끝나서 못쓸 뿐

가격은 7E 유지하고 보존 붙이는 게 강화인 기존 안에다 불가침, 사용시 턴종을 추가하고 에너지 초기화 제거하는 느낌으로 가면 ㄱㅊ을 듯

**Reply by scarf005** [reply]
> 힘깎: ~번 줄입니다 이거 뭐임?

'번' 없앰 오타 ㅈㅅ

> 근대 준비운동 짤은 뭐냐

실수로  내용 까먹음 ㅈㅅ 원래 쓰려고 했던거는  '라이징 스윕' -> '6A' 였음

> 얼티밋 쇼트웨이브: 저게 원작에서 뭐하는 물건인지 모르겠는데, 혹시 설명 가능함? 

원본에선 보호스킬인줄 알았는데 [전멸폭탄](https://youtu.be/8-T8L72dPi4?t=1094)이었음. [갤 의견](https://gall.dcinside.com/mgallery/board/view/?id=slay&no=118901) 보니 에너지 펌핑이 부족하다는 의견이 이 카드도 그렇고 죄다 덱 회전할때만 에너지주는거라 그런것 같음
에너지 펌핑할 방법은 좀 생각해봐야될듯

근데 증폭 무한발동 카드는 뭔지 몰?루 밀리초 펄서말하는거임?

**Reply by 10DUCK100SU** [reply]
카드하나하나 세세하게 의견다는건 힘들고... 

깡방어도 올리는것도 좋지만 코스트 사기치는게 너무 필요함 밀리초펄서,최대출력 말곤 도저히 없음

난 오러리즈선이랑 거대기체행성, 암흑물질 3개가 코스트대비 훌륭하고 시너지도 있는 좋은 방어카드라고 생각하고있음

그밖에 6A,매직앱소버,오르트구름 3개 카드도 좋은것 같은데.. 6A는 머리로는 좋을거 같은데 집기싫음 이유는 모르겠음 ㅋㅋ

나머지는 그냥 나쁘지는 않은데, 공격카드에 증폭을 줘야되니까 방어카드에 코스트를 할당을 못해주니까 생기는 문제라고 생각함

와처 마음의요새 처럼 고능률 방어카드를 주던가, 방어에 증폭을 쓰면 공격에 쓸 코스트가 없어지니 그만큼의 훌륭한 성능을 주면 좋을것 같음

3막까지 가면 솔직히 누가 3이나 4코스트 씀 한턴에 ㅋㅋ 존나 많이쓰지 ㅋㅋ 근데 마리사는 코스트 보충수단이 너무 제한되서 힘듬...

쓰다보니 걍 캐릭을 갈아 엎어야 된다는 방향으로 가는거 같은데... 근데 맞잖아ㅋㅋ 두서없이 써봤는데 3줄요약하면

1. 공격에 증폭하면 방어에 쓸 코스트가 없다
2. 근데 코스트 사기치는 방법이 희귀카드2장 뿐이다
3. 마요나 손절같은 저코 고효율 카드를 줘라 ( 화상태우면 방어도를 주던지, 무감각을 주던지, 방어도 주는 유물을 주던지.... )

**Reply by scarf005** [reply]
ㅇㅋ 방어도랑 코스트위주 개편으로 해보겠음
밸런스패치나 카드수정 아이디어 있으면 [여기서](https://github.com/scarf005/Marisa/discussions/new?category=balance) 부담없이 새 글 써주심 ㄱㅅㄱㅅ

**Reply by 10DUCK100SU** [reply]
뭐 이런 의견이 있다고만 참고하셈... 나도 뉴비라서 딱지겜에 통달한게 아님 ㅋㅋ

걍 하다가 좆같은점을 쓴거 뿐임... 걍 럭키 좆비가 너무 많음

방어카드에 시너지 대충 끼워넣고 방어성능 병신으로 내지 말고 걍 방어는 방어대로 시너지는 시너지대로 다 주는게 맞는거 같음 ( 오러리즈선,거대행성 )

아이언클래드도 기본 4캐릭중에 공격카드가 제일 많은 캐릭인데 방어카드들 성능이 워낙 출중하고, 시너지도 맞추기 쉬워서 전혀 그런 이미지가 없잖음?

어차피 방어카드풀도 적은데 방어도 팍팍줘도 나쁘지 않다고 생각함.. 

그리고 이건 그냥 개인 소견인데 마리사한테 무감각 주는것도 좋아보임...ㅎㅎ

**Reply by 115dkk** [reply]
> > 힘깎: ~번 줄입니다 이거 뭐임?
> 
> '번' 없앰 오타 ㅈㅅ
> 
> > 근대 준비운동 짤은 뭐냐
> 
> 실수로 내용 까먹음 ㅈㅅ 원래 쓰려고 했던거는 '라이징 스윕' -> '6A' 였음
> 
> > 얼티밋 쇼트웨이브: 저게 원작에서 뭐하는 물건인지 모르겠는데, 혹시 설명 가능함?
> 
> 원본에선 보호스킬인줄 알았는데 [전멸폭탄](https://youtu.be/8-T8L72dPi4?t=1094)이었음. [갤 의견](https://gall.dcinside.com/mgallery/board/view/?id=slay&no=118901) 보니 에너지 펌핑이 부족하다는 의견이 이 카드도 그렇고 죄다 덱 회전할때만 에너지주는거라 그런것 같음 에너지 펌핑할 방법은 좀 생각해봐야될듯
> 
> 근데 증폭 무한발동 카드는 뭔지 몰?루 밀리초 펄서말하는거임?

맞음 밀리초펄서임

개인적으로 마리사에게 크게 문제삼는 점 중 하나가 밀리초펄서가 단 한장으로 엄청난 밸류를 내는데다가 아무때나 쎄다는거임. 하다못해 아클 타락도 타수만 있을때는 별로인데 이 미친 카드는 타수만 있어도 기본카드에 좋은 증폭카가 끼어있어서 상당한 성능이 보장됨.

그래서 다른 카드들 버프해주는 대신 밀리초펄서의 너프가 어느정도 이루어져야 하는거 아닌가라는 생각이 듬

**Reply by scarf005** [reply]
결국 펄서 하나로 모든 메타가 굴러가는 근본적인 원인은 증폭 키워드같음
증폭이 적용된 카드 가격이 높게 책정되어있는 반면 대량으로 코스트를 수급할 기회는 밀리초펄서와 최대 출력 둘만 있고
거기에 최대 출력은 소멸 키워드가 있으니 지속적으로 에너지 얻을 수 있는 카드가 펄서 말고는 없음
방어 카드에 더불어 에너지 카드도 개편해야 할것 같음. 4직업 에너지 카드 조사 (전체 카드풀에서 비중, 소멸/지속 여부)해서 비슷하게 맞춰줘야 할것 같음

**Reply by 115dkk** [reply]
> 결국 펄서 하나로 모든 메타가 굴러가는 근본적인 원인은 증폭 키워드같음 증폭이 적용된 카드 가격이 높게 책정되어있는 반면 대량으로 코스트를 수급할 기회는 밀리초펄서와 최대 출력 둘만 있고 거기에 최대 출력은 소멸 키워드가 있으니 지속적으로 에너지 얻을 수 있는 카드가 펄서 말고는 없음 방어 카드에 더불어 에너지 카드도 개편해야 할것 같음. 4직업 에너지 카드 조사 (전체 카드풀에서 비중, 소멸/지속 여부)해서 비슷하게 맞춰줘야 할것 같음

사실 사일/디는 코스트펌핑 운영이 주류지만 아클은 피깎는 사혈 외에는 전부 소멸인 대신 타락으로 코스트를 엄청나게 아끼거나(밀리초펄서의 역할), 또는 잘만 쓰면 코스트를 뛰어넘는 밸류를 얻는 기사회생(+무감각)/참호/한돌/악형 같은 카드로 대체하고 과잉된 방어도는 바리로 붙잡음. 개인적으로 마리사는 아클처럼 직접적인 코스트 카드는 적지만 코스트를 뛰어넘는 밸류를 잘 하면 얻을 수 있는 카드가 많은게 에너지 카드를 늘려 무지성 압축덱을 장려하는 것보다 컨셉상으로 더 유리할거라 생각함

아 그리고 얼티밋 쇼크웨이브 아이디어 떠오름. 완전히 다른 카드가 되지만(특히 이 아이디어는 OP성이 엄청나게 강하니 밸런스 맞추려면 수정이 필요함)

1코 희귀

모든 적을 기절시키고 에너지를 3 얻습니다.
소멸.

증폭 EE(E): 이 카드가 소멸되지 않고 에너지 증가 효과가 1 늘어납니다.

그리고 완전히 잊고 있던 카드가 있는데 이거에 대해서도 피드백함

엄폐

이 카드는 전형적인 독바르기식 카드임. 애매한 밸류에 2코 책정해놓고 강화하면 1코되는 카드

사실상 스택을 뒤집어놓고 뽑을 카드더미를 사실상 초기화하는 카드인데 뽑을 카드 더미 하나 당 1방은 좀 짜다고 생각함

디펙트 합하기가 상향 전에 잘 안쓰인 이유와 비슷한 이유로

그래서 나는 기본 1코로 가고 강화 효과는 방어도를 1->2로 강화하는게 맞지 않나 하고 생각함
